### DISCUSSION TITLE
Inputparameters not used?

### DISCUSSION THREAD

**Discussion by biglimp** [jixuan-chen/target#5] [General]
In the config.ini file there are some settings that I think is not used. Is this correct and if so, what are they used for? Post-processing?

```
domainDim=32,31
latEdge=57.7
lonEdge=12.0
latResolution=0.00088
lonResolution=0.00110
```

**Comment by jixuan-chen** [comment]
domainDim seems unused. I guess it could be used for e.g. saving as tif

The following four can be used to create NetCDF output but I haven't implemented this yet.

latEdge is used in the calculation of UTCI.
